bitwarden_license/src/Commercial.Core/SecretManagerFeatures/Projects/CreateProjectCommand.cs

```diff
@@ -7,7 +7,7 @@ namespace Bit.Commercial.Core.SecretManagerFeatures.Projects;
 public class CreateProjectCommand : ICreateProjectCommand
 {
     private readonly IProjectRepository _projectRepository;
-
+//test
```

Comment:
Looks like we might have forgotten to remove this comment.

Reply:
Removed it, thanks :)